lib/csgo_stats/events/killed.ex

```diff
@@ -1,4 +1,9 @@
 defmodule CsgoStats.Events.Killed do
+  @derive {Jason.Encoder, only: [:killer,
+    :killed,
+    :weapon,
+    :headshot,
+    :penetrated]}
```

Comment:
With how you're rendering it now, I think this can be removed?

Reply:
Yes, left over from testing